### DISCUSSION TITLE
Write segementation masks with Mesmer in 32bit to segement larger ablation areas.

### DISCUSSION THREAD

**Discussion by mabar1** [BodenmillerGroup/steinbock#260] [Q&A]
Dear Steinbock community,
we ablated bigger-than normal IMC tissues with up to 2000x4000px and run into the problem that the 16bit segmentation masks produced via
`steinbock segment deepcell --minmax`
do not have enough bit-depth. I saw the issues [207](https://github.com/BodenmillerGroup/steinbock/issues/207) and [132](https://github.com/BodenmillerGroup/steinbock/issues/132) and I am wondering how to adapt the above command to produce 32bit tiffs.
We are using docker and command line, so I assume one needs to supply `STEINBOCK_MASK_DTYPE` to
`doskey steinbock=docker run -p 8888:8888 -v "pathtodata":/data ghcr.io/bodenmillergroup/steinbock:0.16.1 $*`
Did anyone succeed in creating 32bit segementation maps from Steinbock running in docker?
Best, matthias

**Comment by Milad4849** [comment]
Hi @mabar1 

First off, apologies, the relevant documentation is missing and will be added with the new release. Please modify your docker alias command: `doskey steinbock = docker run -e STEINBOCK_MASK_DTYPE=uint32 ....`

**Reply by mabar1** [reply]
Dear Milad4849,
that is exactly what I needed, thank you very much. I will test it later on. 
Thank you for the fast answer!
